modules/validations-before-agent-iso-creation.adoc

```diff
@@ -12,7 +12,6 @@ is created.
 .`install-config.yaml`
 
 * `baremetal`, `vsphere` and `none` platforms are supported.
-* If `none` is used as a platform, the number of control plane replicas must be `1` and the total number of worker replicas must be `0`.
```

Comment:
I'm deciding to go with Zane's suggestion to outright delete this line, but if it's better to modify the line then I can do that instead.

Reply:
Looks good